services/adminService.js

```diff
@@ -23,36 +25,66 @@ exports.acceptOwnersEmail = async (email, role) => {
   return user
 }
 
-exports.deleteOwner = async (email, role) => {
+exports.deleteOwner = async (id, role) => {
   const user = await User.findOneAndDelete({
-    email: email,
+    _id: id,
     role: role,
```

Comment:
we are deleting owner so we dont need additional argument - role
we can just pass ROLE_OWNER or whatever it is named in app